### DISCUSSION TITLE
How to export a project?

### DISCUSSION THREAD

**Discussion by williamsealy9** [phcode-dev/phoenix#690] [Q&A] [answered]
Hi there

i'm newbie to this field, but i had to edit a html template, now after i finished, i did not find Export option or something, 

your help is appreciated

**Comment by abose** [comment, marked as answer]
Hello @williamsealy9 
1. What is the browser and OS used?
1. Did you edit files in the default project?

**Reply by williamsealy9** [reply]
Hi

1-Chrome, Win7 Ultimate.
2-No i imported a project,not the default.

**Reply by abose** [reply]
Imported from disk? If so the changes r saved in the disk itself.

**Reply by williamsealy9** [reply]
oh thanks a lot

i did't notice that

**Comment by williamsealy9** [comment]
Hi

It's chrome.
No it was a new file ,not the default.
